src/hooks/reviewHooks/useUpdateReviewRequestViewed.ts

```diff
@@ -0,0 +1,19 @@
+import type { AxiosError } from "axios"
+import { useMutation } from "react-query"
+import type { UseMutationResult } from "react-query"
+
+import * as ReviewService from "services/ReviewService"
+
+export const useUpdateReviewRequestViewed = (): UseMutationResult<
+  void,
+  AxiosError<{ message: string }>,
```

Comment:
fyi, not important to fix here, but the error is either `ResponseErrorBody` or, if there are **exceptions**, then `MiddlewareError`.